src/seedu/addressbook/AddressBook.java

```diff
@@ -417,15 +449,15 @@ private static String getMessageForInvalidCommandInput(String userCommand, Strin
      */
     private static String executeAddPerson(String commandArgs) {
         // try decoding a person from the raw args
-        final Optional<String[]> decodeResult = decodePersonFromString(commandArgs);
+        final Optional<String[]> DECODE_RESULT = decodePersonFromString(commandArgs);
```

Comment:
for example in this case, `DECODE_RESULT` is a final object, not a constant (as discussed in class, final objects' properties may be changable if they are not final). hence it should be left to its original naming convention.